lib/src/http_client_response.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+part of '../src/http_client_request.dart';
```

Comment:
> Many Dart developers avoid using part entirely. They find it easier to reason about their code when each library is a single file.

https://dart.dev/guides/language/effective-dart/usage#do-use-strings-in-part-of-directives

Many developers include me. Would you mind just making these files standalone?

Reply:
Yes. Even I don't prefer it :) Is there a way where I can access some variables/functions accross my package without exposing it to the end user?

(I can't completely remember why I moved to part of. I'll give you specifics as I stumble upon them. But, initially, I avoided it.)

Reply:
Does `hide` keyword works for hiding constructors and variables also?

Reply:
Yes, or you can list the things you want to export in https://github.com/unsuitable001/cronet.dart/blob/essentials_v0.0.1/lib/cronet.dart